sample/01-all-features/src/app.module.ts

```diff
@@ -64,6 +65,7 @@ import { user } from '@techsavvyash/user-service';
     MailerModule,
     HomeModule,
     user.UserModule,
+    HealthModule.forRoot(['postgres', 'shadowpostgres', 'fusionauth', 'redis']),
```

Comment:
can we turn these strings into exported constants?

Reply:
Instead of exported constants, shouldn't we pick them from the `.env` itself?

Reply:
I think passing in constants would be a better choice. Here's my reasoning:
1. Having exported types will prevent typos in manual typing and this approach will be more receptive to any internal changes to those constants we might have to make.
2. It is not very often the case when a service depending on other services is not requiring all of them to function. In the event that is actually the case, we can have a second level enable/disable flag from the .env which is not required but optional